airbyte-workers/src/main/java/io/airbyte/workers/WorkerApp.java

```diff
@@ -374,19 +374,12 @@ private static void launchWorkerApp(final Configs configs, final DSLContext conf
     final Path workspaceRoot = configs.getWorkspaceRoot();
     LOGGER.info("workspaceRoot = " + workspaceRoot);
 
-    final String temporalHost = configs.getTemporalHost();
-    LOGGER.info("temporalHost = " + temporalHost);
-
     final SecretsHydrator secretsHydrator = SecretPersistence.getSecretsHydrator(configsDslContext, configs);
 
     if (configs.getWorkerEnvironment().equals(WorkerEnvironment.KUBERNETES)) {
       KubePortManagerSingleton.init(configs.getTemporalWorkerPorts());
     }
 
-    final WorkflowServiceStubs temporalService = TemporalUtils.createTemporalService(temporalHost);
-
-    TemporalUtils.configureTemporalNamespace(temporalService);
```

Comment:
Looks like this line was removed and was not added back anywhere else. Is that intentional? Based on your comment [here](https://github.com/airbytehq/airbyte/pull/13243/files#diff-81a671a2c42eecd8b8ca132bd7b0e518b529eb1cc68867579d2a5450bd38ddb2R123) it sounds like we still want this to be executed for on-prem

Reply:
Yep, meant to add this call back here, thanks for catching!